frontend/packages/ceph-storage-plugin/integration-tests/views/add-capacity.view.ts

```diff
@@ -1,33 +1,39 @@
 import { browser, ExpectedConditions as until, $ } from 'protractor';
 import * as crudView from '@console/internal-integration-tests/views/crud.view';
 import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
-import { click } from '@console/shared/src/test-utils/utils';
+import { click, getOperatorHubCardIndex } from '@console/shared/src/test-utils/utils';
 import { CAPACITY_UNIT, CAPACITY_VALUE, OCS_OP } from '../utils/consts';
 import { currentSelectors } from './installFlow.view';
 
 export const ocsOp = $(`a[data-test-operator-row='${OCS_OP}']`);
-
+export const getStorageClusterLink = async () => {
+  const index = await getOperatorHubCardIndex('Storage Cluster');
+  const link = $(`article:nth-child(${index + 1}) a`);
+  return link;
+};
```

Comment:
I think this has been defined elsewhere.

Reply:
I don't see any. Can you paste the link here?

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts#L63-L66